themes/fll/flltable.css

```diff
@@ -0,0 +1,489 @@
+@import url('fonts/benchnine/benchnine.css');
```

Comment:
This does not work as the font is not there. Also, this styling is visually quite simalar to rednblue-plus, why not add table styling there?

Reply:
The original idea was to have an FLL theme and a more general default theme, with rednblue-plus being the latter. flltable.css is simply a copy-paste of rednblue-plus, with some FLL related things added in (for dealing with the specific sizes of the FLL related sprite images).  That line is not needed in flltable.css, but was just copied over along with the rest of rednblue-plus.css ...